spec/unit/resource_helpers_spec.rb

```diff
@@ -0,0 +1,87 @@
+require_relative 'spec_helper'
+include CernerSplunk::PathHelpers
```

Comment:
Oh I might be a bit off on that, it does work in some places... but I can't refer to the module itself, so the stubs no longer work. I don't know if there is another way to stub these methods.